src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseExpressionBodyForLambda
+{
+    using Microsoft.CodeAnalysis.Text;
```

Comment:
>using [](start = 4, length = 5)

Not sure why some `using`s are inside the namespace

Reply:
fixed.